packages/lib/src/components/Dropin/Dropin.test.ts

```diff
@@ -38,12 +38,14 @@ describe('Dropin', () => {
         });
     });
 
+    // TODO - this test doesn't do anything
```

Comment:
Shall we remove this test then? 😅

Reply:
This was just something I found whilst making my other tests.
I think the correct action would be to actually make it work! But that's out of scope for this PR - I just left this as a reminder